node/core/approval-voting/src/lib.rs

```diff
@@ -101,7 +101,7 @@ pub struct Config {
 // When transitioning from `Syncing` to `Active`, the node notifies the `ApprovalDistribution`
 // subsystem of all unfinalized blocks and the candidates included within them, as well as all
 // votes that the local node itself has cast on candidates within those blocks.
-enum Mode {
+pub enum Mode {
```

Comment:
alternatively, it's possible to create a mock `SyncOracle` which returns `false` on `fn is_major_syncing`

Reply:
Yeah, that'd be a better path, IMO.